src/main/java/uk/gov/hmcts/reform/pdf/service/exception/AuthException.java

```diff
@@ -0,0 +1,8 @@
+package uk.gov.hmcts.reform.pdf.service.exception;
+
+public class AuthException extends RuntimeException {
```

Comment:
Can we not abbreviate this please? **Auth**entication and **auth**orization are different things.

Reply:
Applies to all places with the abbreviation.